test/verify/check-system-realms

```diff
@@ -84,6 +84,15 @@ ExecStart=/bin/true
 
 @testlib.skipDistroPackage()
 class CommonTests:
+
+    def wait_discover(self):
+        with self.browser.wait_timeout(60):
+            self.browser.wait_attr("#realms-op-address", "data-discover", "done")
+
+    def wait_address_helper(self, expected=None):
```

Comment:
This could have been `expected="Contacted domain"` which is a bit nicer I'd say. Anyway not critical.